scipy/stats/_entropy.py

```diff
@@ -27,10 +31,10 @@ def entropy(pk, qk=None, base=None, axis=0):
 
     Parameters
     ----------
-    pk : sequence
+    pk : array_like
         Defines the (discrete) distribution. ``pk[i]`` is the (possibly
```

Comment:
This is the description that should probably be generalized. To avoid potentially unfamiliar NumPy notation for ndarrays, maybe something like 
```
Along each axis-slice of ``pk``, element ``i`` is the  (possibly unnormalized) probability of event ``i``.
```
(with line breaks as needed).

(gh-14680 mentions 
> The entropy function allows calculation of entropy along an axis for multi-variate distributions.

but this is not the only way of thinking of the use of `entropy` on an nd array. For example, each row of a 2d-array could represent a separate univariate distribution.)

Reply:
Mathematically speaking, I think we must consider this (for the case of a 2d array), as a set of univariate RVs. If the distribution is really multivariate then each column (or row) is a realization and not each element on its own.

Can I modify somehow this PR t improve the the description according to your suggestion?

Reply:
Yes. Just push another commit to your branch.